src/ray/raylet/worker_pool.cc

```diff
@@ -138,7 +138,9 @@ void WorkerPool::StartWorkerProcess(const Language &language) {
   int rv = execvp(worker_command_args[0],
                   const_cast<char *const *>(worker_command_args.data()));
   // The worker failed to start. This is a fatal error.
-  RAY_LOG(FATAL) << "Failed to start worker with return value " << rv;
+  if (rv < 0) {
```

Comment:
@jovan-wong I think it's better without the `if (rv < 0)`. The if statement makes it seem like it's ok to hit this line if `rv == 0`. However, if we are ever in this part of the code, we should call `RAY_LOG(FATAL)` even if `rv == 0`.

I think you're saying that's impossible, but it may improve clarity to remove the `if` statement.

What do you think?

Reply:
U r right.  it means something wrong if `exec()` returns a value.
It seems that we don't need `rv` any more.